CHANGES.rst

```diff
@@ -5,7 +5,7 @@ Changelog of hydxlib
 0.2 (unreleased)
 ----------------
 
-- Nothing changed yet.
+- Add docstrings and update readme.rst
```

Comment:
Verleden tijd gebruiken. Dit is wat er in 0.2 veranderd is, het zijn geen instructies aan de gebruiker.

```suggestion
- Added docstrings and updated the readme.
```